docs/docs/recipes/deploying-your-site.md

```diff
@@ -1,132 +1,132 @@
 
-## Preparing for deployment
+## Przygotowanie do wdrożenia
 
-### Prerequisites
+### Wymagania
 
-- A [Gatsby site](/docs/quick-start)
-- The [Gatsby CLI](/docs/gatsby-cli) installed
+- Strona [Gatsby](/docs/quick-start)
+- Zainstalowany [Gatsby CLI](/docs/gatsby-cli)
 
-### Directions
+### Wskazówki
 
-1. Stop your development server if it is running (`Ctrl + C` on your command line in most cases)
+1. Jeśli serwer deweloperski jest uruchomiony, zatrzymaj go (w większości przypadków wystarczy `Ctrl + C` w konsoli)
 
-2. For the standard site path at the root directory (`/`), run `gatsby build` using the Gatsby CLI on the command line. The built files will now be in the `public` folder.
+2. Dla standardowej ścieżki strony z głównym katalogiem (`/`), korzystając z Gatsby CLI uruchom w konsoli `gatsby build`. Zbudowane pliki będą się znajdowały w folderze `public`.
 
 ```shell
 gatsby build
 ```
 
-3. To include a site path other than `/` (such as `/site-name/`), set a path prefix by adding the following to your `gatsby-config.js` and replacing `yourpathprefix` with your desired path prefix:
+3. Aby zawrzeć na stonie inną ścieżkę niż `/` (jak np. `/tytul-strony/`), ustaw przedrostek dodając poniższy kod do swojego `gatsby-config.js` i zamień ścieżkę `yourpathprefix` na swoją własną:
 
 ```js:title=gatsby-config.js
 module.exports = {
   pathPrefix: `/yourpathprefix`,
 }
 ```
 
-There are a few reasons to do this -- for instance, hosting a blog built with Gatsby on a domain with another site not built on Gatsby. The main site would direct to `example.com`, and the Gatsby site with a path prefix could live at `example.com/blog`.
+Istnieje kilka powodów by użyć -- ,dla przykładu hosting bloga zbudowanego za pomocą Gatsby na domenie z inną stroną, która nie jest zbudowana w Gatsby. Główna strona mogłaby przekierowywać do `przyklad.com`, a strona wykorzystująca Gatsby byłaby dostępna pod adresem `przyklad.com/blog`.
 
-4. With a path prefix set in `gatsby-config.js`, run `gatsby build` with the `--prefix-paths` flag to automatically add the prefix to the beginning of all Gatsby site URLs and `<Link>` tags.
+4. Jeśli ustawiłeś przedrostek w `gatsby-config.js`, uruchom `gatsby build` z flagą `--prefix-paths` aby automatycznie dodać przedrostek do początku adresu URL każdej ze stron oraz do tagów `<Link>`.
 
 ```shell
 gatsby build --prefix-paths
 ```
 
-5. Make sure that your site looks the same when running `gatsby build` as with `gatsby develop`. By running `gatsby serve` when you build your site, you can test out (and debug if necessary) the finished product before deploying it live.
+5. Upewnij się, że twoja strona wygląda tak samo używając komendy `gatsby build` jak i `gatsby develop`. Kiedy strona została zbudowana uruchom `gatsby serve`, możesz teraz przetestować jej finalny efekt(i jeśli to potrzebne - poprawić) tuż przed upublicznieniem.
```

Comment:
finalny efekt(i jeśli to potrzebne - poprawić), change to:
finalny efekt (i jeśli to potrzebne - poprawić)
// missing space